Cargo.toml

```diff
@@ -28,14 +28,19 @@ for_test = []
 
 [dev-dependencies]
 fast_image_resize = { path = ".", features = ["for_test"] }
-glassbench = "0.3.3"
 image = "0.24.5"
 resize = "0.7.4"
 rgb = "0.8.34"
 png = "0.17.7"
 nix = { version = "0.26.1", default-features = false, features = ["sched"] }
 testing = { path = "testing" }
+bincode = "1.3.3"
+
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+glassbench = "0.3.3"
 
+[target.'cfg(target_arch = "wasm32")'.dev-dependencies]
+benchmark-simple = "0.1.7"
```

Comment:
Oh, I am trying now to rewrite all of my benches to use of `criterion 0.4`.
The only difficulty I have with `criterion` is how to extract all comparison results to make reports like [benchmarks_x86-64.md](https://github.com/Cykooz/fast_image_resize/blob/main/benchmarks_x86-64.md)

Reply:
Yes, remove it. I am going to finish migration to `criterion` until end of this week.

Reply:
I finished migration to `criterion`. All changes I pushed into `main` branch.